xAPI.md

```diff
@@ -2611,27 +2613,45 @@ can make sense of it.
 ### 4.2 Language Maps
 
 ##### Description
-A language map is a dictionary where the key is a 
-[RFC 5646 Language Tag](http://tools.ietf.org/html/rfc5646), and the value is an 
-string in the language specified in the tag. This map SHOULD be populated as 
-fully as possible based on the knowledge of the string in question in different 
-languages.  
+A language map is a dictionary where the key is a [RFC 5646 Language Tag](http://tools.ietf.org/html/rfc5646), and the value is a string in the language specified in the tag. This map SHOULD be populated as fully as possible based on the knowledge of the string in question in different languages.  There are also places in the specification, 
+such as context, which will require only a language "code" from RFC 5646.
```

Comment:
I prefer to keep the separate ```### 4.2.1 Lang Codes``` section (though it needs an extra #). Language codes are a very different data type from language maps. Language maps contain language codes. 

I'd also be happy just to delete the sentence entirely. I believe we only use lang codes outside of maps within the context and presumably we reference the RFC there.